tests/unit/test_schema_validation.py

```diff
@@ -15,9 +15,17 @@
 import json
 import pytest
 import random
+import re
 from datetime import datetime, timedelta
 
```

Comment:
Can we make use of Python Unittest (https://docs.python.org/3/library/unittest.html) here?

Reply:
We use `pytest` in DVT for unit testing. We've only used `unittest` for mocking objects.